### DISCUSSION TITLE
If another commit is pushed to the same branch, running CI should be cancelled

### DISCUSSION THREAD

**Discussion by EmmanuelMess** [strawberrymusicplayer/strawberry#768] [Ideas]
See [this](https://stackoverflow.com/a/67939898/3124150). Code:
```
concurrency: 
  group: ${{ github.head_ref }}
  cancel-in-progress: true
```

**Comment by jonaski** [comment]
I'm not sure I want that.

**Reply by EmmanuelMess** [reply]
Why not?

**Reply by jonaski** [reply]
Because I like specifically see which commits succeeds or fails when I commit to master.

**Reply by EmmanuelMess** [reply]
> Because I like specifically see which commits succeeds or fails when I commit to master.

Ah!

**Comment by jonaski** [comment]
@EmmanuelMess 
I move this issue to the discussions since it's not a bug.
